_caseStudies/hrgo.html

```diff
@@ -1,19 +1,24 @@
 ---
 layout: default
-title: HR GO Case Study
 sitemap: false
+thumbnail: "/assets/images/work/hrgoThumbnail.jpg"
+title: HRGO.co.uk
+published: "y"
 ---
 
-<div class="banner banner-large" style="background-image:url(/assets/images/blog/happyLuke.jpg)"></div>
-
-<div class="row">
-
-	<div class="columns small-12">
-		<h1>HR GO Case Study</h1>
-
-		<p>Will be the case study for HR GO.</p>
+<section class="section">
+	<div class="row">
+		<div class="columns small-12 medium-6 large-4">
+			<h2>HR GO</h2>
+			<p>
+				We built a living site that internal staff could easily update and change at their whim. We focused on getting the HR GO brand across to customers and giving them the information and confidence to sign up for a job with the company.
+			</p>
+			<p>
+				One of the major objectives on this project was to streamline the sign-up process. We trimmed the sign-up form and changed the user experience to make it easier, quicker and hassle free for the user. We believe that the people make the company and if you have great people you should sell those to the customer! With that in mind, we've added branch pages with each team member at the branch.
+			</p>
+		</div>
+		<div class="columns small-12 medium-6 large-8">
+			<img alt="Screen shots of HR GO on multiple devices" src="/assets/hrGODesktop.png">
```

Comment:
Could we change this for a slightly better image? Maybe using the [locations](http://www.hrgo.co.uk/locations) page